### PR TITLE
Crealogix prestep errors

### PR DESCRIPTION
**Pull request by berezkin88**

### CODE REVIEW

File: adapters/crealogix-adapter/src/main/java/de/adorsys/xs2a/adapter/crealogix/CrealogixAccountInformationService.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+import de.adorsys.xs2a.adapter.api.model.ConsentsResponse201;
+import de.adorsys.xs2a.adapter.impl.BaseAccountInformationService;
+
+import static de.adorsys.xs2a.adapter.crealogix.CrealogixResponseHandlers.CREALOGIX_ERROR_RESPONSE_INSTANCE;
+import static de.adorsys.xs2a.adapter.crealogix.CrealogixResponseHandlers.crealogixResponseHandler;
+import static java.util.function.Function.identity;
+
+public class CrealogixAccountInformationService extends BaseAccountInformationService {
+
+    public static final String ERROR_MESSAGE = "Authorization header is missing or embedded pre-authorization is needed";
```
Comment (spetrychenko) [inline adapters/crealogix-adapter/src/main/java/de/adorsys/xs2a/adapter/crealogix/CrealogixAccountInformationService.java:20]: can we move this message somewhere and reuse it for AIS and PIS services?

File: adapters/crealogix-adapter/src/main/java/de/adorsys/xs2a/adapter/crealogix/CrealogixAccountInformationService.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+                                              HttpClient httpClient,
+                                              LinksRewriter linksRewriter) {
+        super(aspsp, httpClient, linksRewriter);
+    }
+
+    @Override
+    public Response<ConsentsResponse201> createConsent(RequestHeaders requestHeaders,
+                                                       RequestParams requestParams,
+                                                       Consents body) {
+        if (!requestHeaders.get(RequestHeaders.AUTHORIZATION).isPresent()) {
```
Comment (spetrychenko) [inline adapters/crealogix-adapter/src/main/java/de/adorsys/xs2a/adapter/crealogix/CrealogixAccountInformationService.java:32]: can we move this method somewhere and reuse it for avoiding duplication?

Comment (berezkin88) [reply adapters/crealogix-adapter/src/main/java/de/adorsys/xs2a/adapter/crealogix/CrealogixAccountInformationService.java]: rebuild

File: adapters/crealogix-adapter/src/main/java/de/adorsys/xs2a/adapter/crealogix/CrealogixAccountInformationService.java (hunk 1 above)
Comment (berezkin88) [reply adapters/crealogix-adapter/src/main/java/de/adorsys/xs2a/adapter/crealogix/CrealogixAccountInformationService.java]: rebuild

**Review by spetrychenko — APPROVED** [review]

**Pull request merged**